src/Component/Templates/Home.js

```diff
@@ -0,0 +1,114 @@
+import React, {useEffect} from 'react'
+import AddItem from '../Organisms/AddItem'
+import ItemList from '../Organisms/ItemList'
+import TableList from '../Organisms/TableList'
+import styled, { ThemeProvider } from 'styled-components'
+import { DragDropContext } from 'react-beautiful-dnd'
+import { sort,getState } from '../../Store/Action/itemAction';
+import { useDispatch } from 'react-redux';
+import NavBar from '../Molecules/Navbar';
+import { createMuiTheme } from '@material-ui/core/styles';
+import { Typography } from '@material-ui/core'
+
```

Comment:
Template should just have the layout and render the components as props. It should not have any logic.
It should get the components from pages as props.